src/dmd/dmangle.d

```diff
@@ -225,13 +223,13 @@ public:
     {
         if (!t.isTypeBasic())
         {
-            auto p = cast(size_t*)dmd_aaGet(&types, cast(Key)t);
-            if (*p)
+            auto p = cast(void*)t in types;
+            if (p)
```

Comment:
Blasts (to put it nicely).  I've just realised why its failing. (check should be `p && *p` :-)